src/runtime/LoadingView.tsx

```diff
@@ -17,31 +18,37 @@ export default class LoadingView extends React.Component<Props> {
       const error = value["_error"].message;
 
       // make sure the error isn't a duplicate
-      const found = this.errors.findIndex((err) => 
-        err === error
-      ) > -1;
+      const found = this.errors.findIndex((err) => err === error) > -1;
 
       if (!found) {
         this.errors.push(error);
       }
     }
-  }
+  };
 
   public render() {
     const { logs } = this.props;
 
     R.forEachObjIndexed(this.findErrorKeys, logs);
     return (
-      <Popup
-        trigger={<Spinner name='double-bounce'/>}
-        position="right center"
-        on="hover"
-        contentStyle={{ width: "auto", display: "flex", flexWrap: "wrap" }}
-      >
-        <div>
-          {this.errors.map(error => error)}
-        </div>
-      </Popup>
-    )
+      <Loader.Render>
```

Comment:
Also, I was wondering if should we render the loader here or here https://github.com/dOrgTech/DAOcomponents/blob/master/src/runtime/ContextFeed.tsx#L68

Reply:
No, we want to use LoaderView inside of run-time components, then LoaderView decides whether to use the default loader or the user-specified loader.

LoaderView = internal loading renderer
Loader = external loading config